pymc4/inference/utils.py

```diff
@@ -18,3 +19,26 @@ def initialize_state(model: Model, observed: Optional[dict] = None) -> flow.Samp
     """
     _, state = flow.evaluate_model_transformed(model, observed=observed)
     return state.as_sampling_state()
+
+
+def trace_to_arviz(pm4_trace, pm4_sample_stats):
+    """
+    Tensorflow to Arviz trace convertor.
+
+    Convert a PyMC4 trace as returned by sample() to an ArviZ trace object
```

Comment:
Would change this to to an `az.InferenceData object` or `Arviz InferenceData object`

Reply:
I am not sure, this is a bit too specific. Let me try to see if I can add a helper class in TFP so that the output is a bit more standardized.

Reply:
Add types (including return type)?  This information seems to be available in the docstrings...